sycl/test-e2e/BFloat16/bfloat16_builtins_cuda_generic.cpp

```diff
@@ -5,6 +5,7 @@
 // + below sm_80 always uses generic impls
 
 // DEFINE: %{mathflags} = %if cl_options %{/clang:-fno-fast-math%} %else %{-fno-fast-math%}
+// REQUIRES: aspect-ext_oneapi_bfloat16_math_functions
```

Comment:
There is an open draft that attempts to remove that aspect: #13351 and it seems weird to have another PR that tries to use it. Tagging @JackAKirk here to comment

Reply:
Makes sense, but I would lean towards having all of the tests using that aspect be consistent, and when that draft is ready to merge it can update all of the tests using it.

Reply:
It would be better to just remove the aspect here and in the other .cpp , and then xfail cpu in these tests instead: to match the way that the other bfloat16 tests are run (all bfloat16 tests that fail on cpu backend, including this test, are failing for the same reason: opaque pointer issue). aspect-ext_oneapi_bfloat16_math_functions has no purpose and is misleading. All these bfloat16 tests should be supported on all backends. opencl cpu support was broken for some devices when the switch to opaque pointers was made, and I believe that a fix for this is being looked into.

Reply:
Makes sense; I've done that now